src/rpc/Errors.cpp

```diff
@@ -89,6 +89,11 @@ getErrorInfo(ClioError code)
         {.code = ClioError::RpcMalformedAuthorizedCredentials,
          .error = "malformedAuthorizedCredentials",
          .message = "Malformed authorized credentials."},
+        {.code = ClioError::RpcMalformedObject, .error = "malformedObject", .message = "Malformed object."},
+        {.code = ClioError::RpcMalformedObjectId, .error = "malformedObjectId", .message = "Malformed object ID."},
+        {.code = ClioError::RpcMalformedAccount, .error = "malformedAccount", .message = "Malformed account."},
```

Comment:
Maybe we can add //TODO here: replace when rippled side adds the error code. I think these errors should be added at rippled side.

Reply:
The PR https://github.com/XRPLF/rippled/pull/5252 in rippled was merged, so I will remove new errors here.